### DISCUSSION TITLE
Duke Nukem Megaton Edition?

### DISCUSSION THREAD

**Discussion by dm5kmwf** [community/community#21401] [Other Feature Feedback, Questions, & Ideas]
<p>Sorry if this is in the wrong section. I downloaded the Duke Nukem Megaton Edition below and I’m not sure how to launch the game. Do I need the original game to place this in? Or a program like GZ Doom? Any help would be appreciated.</p>
<p><a href="https://github.com/TermiT/duke3d-megaton" rel="nofollow noopener">https://github.com/TermiT/duke3d-megaton</a></p>

**Comment by MChevy422** [comment]
<p>Hello <a class="mention" href="/dm5kmwf">@dm5kmwf</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the ‘Programming Help &amp; Discussion’ board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Have you taken a look at <a href="https://github.com/coelckers/gzdoom" title="https://github.com/coelckers/gzdoom" rel="nofollow noopener">this?</a></p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Greatly appreciated,</p>
<p><strong>-MChevy422</strong></p>

**Comment by fire-eggs** [comment, marked as answer]
<p>The repository contains the source code, not binaries. You’ll need to get the project built - Visual Studio Express 2008 or later should do the trick.</p>
